### PR TITLE
Android sdk 5.12.2

### PR DESCRIPTION
**Pull request by zvs001**
It is update for https://github.com/mieszko4/react-native-zoom-us/pull/219
Note that I selected sub branch for update.

I only added android sdk 5.12.2  and merged master.

Issue with `__emutls_get_address` (https://github.com/mieszko4/react-native-zoom-us/pull/219) is still exists.

### CODE REVIEW

File: android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java [hunk 1]
```diff
@@ -1095,11 +1108,13 @@ public void onFreeMeetingNeedToUpgrade(FreeMeetingNeedUpgradeType type, String g
...
-  public void onAllHandsLowered() {};
+  public void onAllHandsLowered() {}
+  @Override
+  public void onPermissionRequested(String[] permissions) {}
   @Override
-  public void onPermissionRequested(String[] permissions) {};
+  public void onChatMsgDeleteNotification(String msgID, ChatMessageDeleteType deleteBy) {}
   @Override
-  public void onChatMsgDeleteNotification(String msgID, ChatMessageDeleteType deleteBy) {};
+  public void onShareMeetingChatStatusChanged(boolean start) {}
```
Comment (mieszko4) [inline android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java:1117]: Looks like this is the only actual change of this file

File: android/build.gradle [hunk 2]
```diff
@@ -35,7 +35,7 @@ android {
 
 dependencies {
     implementation 'com.facebook.react:react-native:+' // From node_modules
-    implementation 'com.github.zoom-us-community:jitpack-zoom-us:5.11.0.6883'
+    implementation 'com.github.zoom-us-community:jitpack-zoom-us:5.12.2.9109'
```
Comment (mieszko4) [inline android/build.gradle:38]: Looks like this is the only actual change of this file

**Review by mieszko4 — COMMENTED** [review]
Thnx @zvs001 !
This PR has been showing a lot of irrelevant changes.
I've just merged `master` into `feat/upgrade-android-to-5.11.0.6883-2` branch but still the same issue.

Could you rebase your branch or create a new PR so that we can see the actual changes?

**Comment by zvs001** [comment]
Yes. I will update tomorrow

**Review by mieszko4 — APPROVED** [review]
:+1:

**Pull request merged**